src/frontend/core/CoreConstants.cpp

```diff
@@ -0,0 +1,14 @@
+/**
+Copyright 2019 JasminGraph Team
```

Comment:
Year of the copyright has to change to `2021`

Reply:
Change `JasminGraph` to `JasmineGraph`

Reply:
File was changed as requested